worksheets.go

```diff
@@ -93,6 +93,8 @@ func NewDefinitions(reader io.Reader, opts ...Options) (*Definitions, error) {
 			// Any bad index?
 			if field.index == 0 {
 				return nil, fmt.Errorf("%s.%s: index cannot be zero", def.name, field.name)
+			} else if field.index > 65536 {
```

Comment:
I think most code-readers will understand the significance of `65536`, but it might be more literate to express this as `math.MaxUint16 + 1`, so your intent is clear. Same thing for other instances of this.